xlators/debug/io-stats/src/io-stats.c

```diff
@@ -1398,6 +1404,19 @@ io_stats_dump_global_to_logfp(xlator_t *this, struct ios_global_stats *stats,
                     gf_upcall_list[i], fop_hits, "0", "0", "0");
     }
 
+    /* To measure FOP_HITS_PER_SEC consider time difference
+       between last incremental profile stat started and last
+       fop hit.
+    */
+    duration = (conf->last_fop_hit - stats->started_at);
+    if (duration > 0) {
+        ios_log(this, logfp,
+                "\n"
```

Comment:
Is this really necessary ?

Reply:
With a new line, it is showing clearly.

Reply:
> But doesn't this log a line that doesn't start with the standard date and time, log level, ... ?

Now I am not calling with a newline, I did already remove it.